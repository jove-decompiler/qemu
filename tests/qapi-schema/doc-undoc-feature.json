# Doc comment must cover all features
# BUG: missing documentation for @undoc not caught

##
# @foo:
#
# Features:
# @doc: documented feature
##
{ 'command': 'foo', 'features': ['undoc', 'doc'] }
