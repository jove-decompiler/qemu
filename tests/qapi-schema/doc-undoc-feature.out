module None
object q_empty
enum QType
    prefix QTYPE
    member none
    member qnull
    member qnum
    member qstring
    member qdict
    member qlist
    member qbool
module doc-undoc-feature.json
command foo None -> None
    gen=True success_response=True boxed=False oob=False preconfig=False
    feature undoc
    feature doc
doc symbol=foo
    body=

    feature=doc
documented feature
